components/application-mgt/org.wso2.carbon.identity.application.common/src/main/java/org/wso2/carbon/identity/application/common/model/ServiceProvider.java

```diff
@@ -53,6 +53,7 @@ public class ServiceProvider implements Serializable {
     private static final String IS_MANAGEMENT_APP = "IsManagementApp";
 
     private static final String IS_B2B_SELF_SERVICE_APP = "IsB2BSelfServiceApp";
+    private static final String APP_ACCESS_ENABLED = "applicationAccessEnabled";
```

Comment:
```suggestion
    private static final String IS_APP_ACCESS_ENABLED = "isApplicationAccessEnabled";
```
Shall we renamed to keep consistency with other configs?